config/configmap/instascale_configmap.yaml

```diff
@@ -0,0 +1,7 @@
+kind: ConfigMap
```

Comment:
A convention with Kustomize is to name some directories according the target deployment environment. So there could be a directory called `local` that would import resources from `default` and add the ConfigMap. Another alternative would be to add the ConfigMap to the `default` directory. The idea being only one `kustomize build` command is necessary.

Reply:
Yes, this would be my preference too. 
The current kustomize build adds the default deployments to `instascale-1-system` namspace. With the prefix `instascale-1`.

This will lead to the config map being deployed to `instascale-1-system' under the name `instascale-1-instascale-config`.

I wanted to ask if there is an alternative approach to this, or if that would be suitable?

Reply:
IMO, having the `instascale-config` within the `kube-system` namespace doesn't really make sense.

Usually, the convention is to have controllers / operators installing all their resources in their own namespace.

So my suggestion would be to deploy everything inside the `codeflare-system`, choosing `codeflare` so other component like MCAD can be deployed there also.

Reply:
Will [this](https://github.com/project-codeflare/codeflare-operator/issues/273) make this config map redundant? will we be pulling in values from a single CFO config map?

Reply:
This ConfigMap will be used for local development. The CFO ConfigMap will be the union of all the component configurations.